### PR TITLE
Feature/contact page

### PR DESCRIPTION
**Pull request by nicolecaparas**
Create contact page form 
Fix hover button: 
1. Landing 
2. Navigation (Reg/ Login)

### CODE REVIEW

File: src/Contact.jsx [hunk 1]
```diff
@@ -2,7 +2,46 @@ import React from 'react'
 
 function Contact () {  
     return (
-        <div></div>
+        <div class="bg-white text-gray-100 px-8 py-12">
+              <div
+                class="max-w-screen-xl mt-5 px-8 grid gap-8 grid-cols-1 md:grid-cols-2 md:px-12 lg:px-16 xl:px-32 py-10 mx-auto bg-gray-100 text-gray-900 rounded-lg shadow-lg">
+                <div class="flex flex-col justify-between">
+                  <div>
+                    <h2 class="text-4xl lg:text-5xl font-bold leading-tight">Lets talk about everything!</h2>
```
Comment (jpcaparas) [inline src/Contact.jsx]: Spelling, should be `Let's`

**Comment by jpcaparas** [comment]
Looks good to me, but just left a minor comment; resolve that, I'll approve.

**Comment by nicolecaparas** [comment]
Correct spelling

**Comment by jpcaparas** [comment]
Looks good

**Review by jpcaparas — APPROVED** [review]
Happy with the changes

**Pull request merged**